lib5k/src/main/java/io/github/frc5024/purepursuit/pathgen/BezierPath.java

```diff
@@ -0,0 +1,214 @@
+package io.github.frc5024.purepursuit.pathgen;
+
+import java.util.ArrayList;
+import ca.retrylife.ewmath.MathUtils;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.util.Units;
+
+/**
+ * This class is used to generate points along a bezier curves
+ */
+public class BezierPath extends Path{
+
+	// Maximum space allowed between points
+	private double maxSeperation = Units.inchesToMeters(4);
+
+	// The lookup table
+	private static ArrayList<int[]> binomialLookUpTable;
+
+	/**
+	 * 
+	 * @param wayPoints The 3 way points of a quadratic bezier curve
+	 */
+	public BezierPath(Translation2d[] wayPoints) {
```

Comment:
Since you are manually setting 3 weights, can you make this function look like this? 

`public BezierPath(Translation2d start, Translation2d middle, Translation2d end)`